pkgs/applications/science/electronics/alliance/default.nix

```diff
@@ -1,74 +1,46 @@
-{ lib, stdenv, fetchurl
-, xorgproto, motif, libX11, libXt, libXpm, bison
-, flex, automake, autoconf, libtool, runtimeShell
+{ lib, stdenv, fetchFromGitLab, xorgproto, motif, libX11, libXt, libXpm, bison
+, flex, automake, autoconf, libtool
 }:
 
 stdenv.mkDerivation rec {
   pname = "alliance";
-  version = "5.1.1";
-
-  src = fetchurl {
-    url = "http://www-asim.lip6.fr/pub/alliance/distribution/5.0/${pname}-${version}.tar.bz2";
-    sha256 = "046c9qwl1vbww0ljm4xyxf5jpz9nq62b2q0wdz9xjimgh4c207w1";
+  version = "unstable-2020-12-04";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.lip6.fr";
+    owner = "vlsi-eda";
+    repo = "alliance";
+    rev = "c12f377989571c1d868bebebdfb1574572e2ccf9";
+    sha256 = "RNfW621hRx5pzg5GhtY4sG8jwuQws3oG8EjZF7Mt5B0=";
   };
 
+  prePatch = "cd alliance/src";
 
   nativeBuildInputs = [ libtool automake autoconf flex ];
   buildInputs = [ xorgproto motif libX11 libXt libXpm bison ];
 
-  sourceRoot = "alliance/src/";
+  ALLIANCE_TOP = placeholder "out";
 
   configureFlags = [
-    "--prefix=$(out)"
+    "--prefix=${placeholder "out"}" "--enable-alc-shared"
   ];
 
-  preConfigure = ''
-    mkdir -p $out/etc
-
-    #texlive for docs seems extreme
-    mkdir -p $out/share/alliance
-    mv ./documentation $out/share/alliance
+  postPatch = ''
+    # texlive for docs seems extreme
```

Comment:
you kept the comment but removed the documentation, is this intended?

Reply:
Yes, the documentation is still not built due to the `--replace`.